components/avatar/avatar.css

```diff
@@ -0,0 +1,44 @@
+.avatar {
+    margin: 1rem;
+    border-radius: 50%;
+    width: 50px;
+    height: 50px;
+    position: relative;
+    display: inline;
+}
+
+.figure {
+    border-radius: 50%;
+}
```

Comment:
I think this class can be removed since it's being used with `avatar` which already has `border-radius: 50%`.